app/models/zip_code_search.rb

```diff
@@ -8,7 +8,7 @@ def error_message
   end
 
   def query
-    search_params[:query].rjust(5, '0')
+    search_params[:query]
```

Comment:
This was vestigial code from the era where the search box was a `<input type="number">` and leading zeroes weren't passed along.